tests/test_dse.py

```diff
@@ -1299,11 +1299,11 @@ def test_fullopt(self):
         assert len(op0._func_table) == 0
         assert len(op1._func_table) == 1  # due to loop blocking
 
-        assert summary0[('section0', None)].ops == 39
-        assert np.isclose(summary0[('section0', None)].oi, 2.224, atol=0.001)
+        assert summary0[('section0', None)].ops == 46
```

Comment:
39 -> 46 is a bit suspicious, as it's roughly 15%. What kind of operations we now have ? is it just +/* or is it divisions? can you copy-paste the stencil expressions somewhere (or I'll take a look no worries)

Reply:
TTI opcounts are fine (actually slightly lower) so I think the compiler is fine. THis is the "noop" case so only mean the basic solution of solve is not as simplified as it was doing it by hand.